CMakeLists.txt

```diff
@@ -304,7 +367,8 @@ if(DESKTOP)
     )
     target_link_libraries(libuWS
       PRIVATE
-        ${OPENSSL_LIBRARIES}
+        OpenSSL::SSL
```

Comment:
Since ssl->crypto dependency is captured already in ssl, do we need to specify these 2 libraries separately here?